src/exco/extractor/cell_extraction_task.py

```diff
@@ -91,15 +92,25 @@ def __str__(self):
         parser: {self.parser}
         validators: {[dict(key=key, v=v) for key, v in self.validators.items()]}
         assumptions: {[dict(key=key, a=a) for key, a in self.assumptions.items()]}
-        fallback: {self.fallback}""")
+        fallback: {self.fallback}
+        deref: {self.deref}
+        """)
         return s
 
     def process(self, anchor_cell_location: CellLocation,
                 workbook: Workbook) -> CellExtractionTaskResult[T]:
+        if self.deref:
```

Comment:
mutating state here is a bad idea. The object can't be reused predictably a' and even worse in multithreaded environment.